ports/mimxrt/hal/startup_MIMXRT10xx.S

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
I'd call this file `resethandler_MIMXRT10xx.S`.  Also it'll need copyright/license copied from the file it was taken from, because there's a lot of copied code.